src/pages/settings/Wallet/PaymentMethodList.tsx

```diff
@@ -190,20 +191,13 @@ function PaymentMethodList({
         if (shouldShowAssignedCards) {
             const assignedCards = Object.values(cardList ?? {})
                 // Filter by physical, active cards associated with a domain
-                .filter((card) => !card.isVirtual && !!card.domainName && CONST.EXPENSIFY_CARD.ACTIVE_STATES.includes(card.state ?? 0))
-                .toSorted((card1, card2) => {
-                    const isExpensifyCard1 = CardUtils.isExpensifyCard(card1.cardID);
-                    const isExpensifyCard2 = CardUtils.isExpensifyCard(card2.cardID);
-                    if (isExpensifyCard1 === isExpensifyCard2) {
-                        return 0;
-                    }
-
-                    return isExpensifyCard1 ? -1 : 1;
-                });
+                .filter((card) => !card.isVirtual && !!card.domainName && CONST.EXPENSIFY_CARD.ACTIVE_STATES.includes(card.state ?? 0));
 
             const numberPhysicalExpensifyCards = assignedCards.filter((card) => CardUtils.isExpensifyCard(card.cardID)).length;
 
-            return assignedCards.map((card) => {
+            const assignedCardsSorted = _.sortBy(assignedCards, (card) => !CardUtils.isExpensifyCard(card.cardID));
```

Comment:
I see your comment in original PR. I am not against using lodash as long as lint success.
Please `import lodashSortBy from 'lodash/sortBy';`

Reply:
> We prefer native js method to lodash in TS

Yeah, there's no native equivalent, and using`sort` leads to awful bugs (not in theory; in practice). I tried polyfilling `toSorted` but had some weird problem with ESLint (unrelated to the change itself, likely because of the changes in the NPM modules).

Reply:
> But the linter is fine. Do you still prefer `import lodashSortBy from 'lodash/sortBy';` even when "Lint code" check succeeded?

@blazejkustra or @fabioh8010 may have answer

Reply:
I switched to `lodashSortBy`

Reply:
`lodashSortBy` is fine 👍🏻